zip-1014.rst

```diff
@@ -144,37 +154,36 @@ address(es) for each block. This Dev Fund will end at the second halving (unless
 extended/modified by a future ZIP).
 
 
-ECC slice (Electric Coin Company)
+BP slice (Bootstrap Project)
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-This slice of the Dev Fund will flow to ECC.
-
-ECC MUST undertake a firm obligation to use the Dev Fund only in support of the
-Zcash cryptocurrency and its community.
-
-In particular, ECC MUST commit to not distribute the Dev Fund proceeds to its
-partners ("shareholders"), other than:
-
-1. In fair-market-value compensation for specific new work (e.g., to employees
-   and contractors).
-2. For covering pass-through tax obligations to partners caused by ECC's receipt
-   of the Dev Fund.
-
-(ECC is encouraged to transition to a corporate structure that would avoid the
-latter taxes.)
-
-This obligation MUST be made irrevocable, e.g., within ECC's corporate
-governance structure (i.e., its Operating Agreement) or contractual obligations.
+This slice of the Dev Fund will flow as charitable contributions from
+the Zcash Community to the Bootstrap Project, the newly formed parent
+organization to the Electric Coin Company.  Bootstrap Project is organized
+for exempt educational, charitable, and scientific purposes in
+compliance with Section 501(c)(3) of the Code, including but not
```

Comment:
```suggestion
compliance with Section 501(c)(3), including but not
```

Reply:
If the document refers to "Section 501(c)(3) of the US Code", that's nicely specific and unambiguous, but also a mouthful, so why not just clarify once at the initial reference, then use a shorthand elsewhere?

Reply:
Again, addressed by https://github.com/zcash/zips/pull/409/files#r508559299